config/routes.rb

```diff
@@ -2,4 +2,10 @@
   resources :positions
   resources :employees
   resources :departments
+  resources :employees do
```

Comment:
Clear up the routes file. No need for creating additional resources for `employees` when you want nested resources.
```ruby
resources :employess do
  resources :position_histories, only: [:new, :create, :edit, :update]
  resources :vacations, only: [:new, :create]
end
```